### PR TITLE
[DE] Add `History of osu!/Online rankings/osu!`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Most of this article is tables, so it's less than it looks.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/Online_rankings/osu!/de.md [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+| ::{ flag=US }:: [Freddie Benson](https://osu.ppy.sh/users/7342622) | 13 |
+| ::{ flag=JP }:: [tobebuta](https://osu.ppy.sh/users/14053) | 13 |
+| ::{ flag=JP }:: [Flute](https://osu.ppy.sh/users/211278) | 8 |
+| ::{ flag=KR }:: [La Valse](https://osu.ppy.sh/users/70863) | 7 |
+| ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) | 6 |
+| ::{ flag=TW }:: [f0012979](https://osu.ppy.sh/users/12143) | 4 |
+| ::{ flag=CN }:: [5231_Kinoko](https://osu.ppy.sh/users/181057) | 1 |
+| ::{ flag=FI }:: [thelewa](https://osu.ppy.sh/users/475021) | 1 |
+
+## Längste ununterbrochene Titelverteidigung
```
Comment (The-Last-Cookie) [inline wiki/History_of_osu!/Online_rankings/osu!/de.md:371]: A few alternatives that came to my mind:

- `durchgehend`
- `fortlaufend`
- `kontinuierlich`
- `zusammenhängend`

Though I prefer the current version. It emphasises that there was no *break* in-between.

File: wiki/History_of_osu!/Online_rankings/osu!/de.md [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+- [ppv1](/wiki/Performance_points/ppv1) (vom 24. Juli 2012 bis 27. Januar 2014)
+- [ppv2](/wiki/Performance_points) (vom 27. Januar 2014 bis heute)
+
+Darum enthält diese Liste nur Spieler, die den Platz 1 auf der Hauptrangliste erreichten, die zu diesem Zeitpunkt verwendet wurde. Daten und Zahlen *in Kursivschrift* geben an, dass der entsprechende Spieler seinen Titel als Gewinner von osu! weiterhin entweder im vorhergehenden oder im folgenden Jahr souverän verteidigt.
+
+Es wird zwar angenommen, dass ::{ flag=AU }:: [peppy](https://osu.ppy.sh/users/2) der erste Spieler auf Platz 1 war, aber keine Quelle kann diese Behauptung belegen.
+
+## 2007
+
+*Anmerkung: In der Zeit, in der [Ranked Score](/wiki/Gameplay/Score/Ranked_score) im Einsatz war, wechselte der Spieler auf dem ersten Platz häufig. Daher sind die früheren Einträge ungenauer. Detaillierte Informationen können in der voraussichtlichen Rekonstruktion der globalen Rangliste von Ranked Score gefunden werden, die [hier](https://docs.google.com/spreadsheets/d/1fcFtTNim7hminC2LaMGwTBwa6_GIHU8Sz-wQ_eXymiE) zur Verfügung steht.*
```
Comment (respektive) [inline wiki/History_of_osu!/Online_rankings/osu!/de.md:30]:
```suggestion
*Anmerkung: In der Zeit, in der [Ranked Score](/wiki/Gameplay/Score/Ranked_score) im Einsatz war, wechselte der Spieler auf dem ersten Platz häufig. Daher sind die früheren Einträge ungenauer. Detaillierte Informationen können in der geschätzten Rekonstruktion der globalen Rangliste von Ranked Score gefunden werden, die [hier](https://docs.google.com/spreadsheets/d/1fcFtTNim7hminC2LaMGwTBwa6_GIHU8Sz-wQ_eXymiE) zur Verfügung steht.*
```

File: wiki/History_of_osu!/Online_rankings/osu!/de.md [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+| ::{ flag=PL }:: [WubWoofWolf](https://osu.ppy.sh/users/39828) | 24.11.2013 | *20.01.2014* | [\[1\]](https://old.reddit.com/r/osugame/comments/1rctoz/wubwoofwolf_went_on_a_little_rampage/) |
+
+## 2014
+
+*Anmerkung: [ppv2](/wiki/Performance_points) wurde am 27. Januar 2014 eingeführt.*
+
+| Spieler | von | bis | Quellen |
+| --: | :-- | :-- | :-- |
+| ::{ flag=PL }:: [WubWoofWolf](https://osu.ppy.sh/users/39828) | *24.11.2013* | 20.01.2014 | [\[1\]](https://old.reddit.com/r/osugame/comments/1rctoz/wubwoofwolf_went_on_a_little_rampage/) |
+| *vakant* | 20.01.2014 | 27.01.2014 | [\[1\]](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)  |
```
Comment (respektive) [inline wiki/History_of_osu!/Online_rankings/osu!/de.md:166]: maybe
```suggestion
| *unbesetzt* | 20.01.2014 | 27.01.2014 | [\[1\]](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)  |
```

File: wiki/History_of_osu!/Online_rankings/osu!/de.md (hunk 1 above)
Comment (respektive) [reply wiki/History_of_osu!/Online_rankings/osu!/de.md:371]: Current one is good yeah

**Review by respektive — CHANGES_REQUESTED** [review]

**Review by respektive — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**